### DISCUSSION TITLE
Re-image vs upgrade

### DISCUSSION THREAD

**Discussion by davidstoll** [Joshua-Riek/ubuntu-rockchip#169] [General]
It has probably been stated somewhere, but maybe I'm not searching with the right key words.  I was just wondering if I can just update/upgrade this rockchip Ubuntu vs downloading the latest image and just blowing everything away...?

**Comment by Joshua-Riek** [comment]
The latest release is a bit different because of massive kernel changes. There are about 6000 new code commits from Rockchip.

Normally, you can easily upgrade with a simple apt upgrade command. However, due to the massive kernel changes, you will need to re-install the system.

There is a way to update the kernel if you are using v1.15, but it's not as simple as a single apt upgrade command. I will post a guide sometime later.

**Reply by 0asisX** [reply]
> The latest release is a bit different because of massive kernel changes. There are about 6000 new code commits from Rockchip.
> 
> Normally, you can easily upgrade with a simple apt upgrade command. However, due to the massive kernel changes, you will need to re-install the system.
> 
> There is a way to update the kernel if you are using v1.15, but it's not as simple as a single apt upgrade command. I will post a guide sometime later.

Hey, are you still planning to post it? Would be happy to upgrade to latest without re-image. Thanks!
